### PR TITLE
Fix accidentally bold h-cards

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._ 

- bird bios on list & detail pages
- citations in quotes
- copyright info in footer

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Oops, About and Author pages are all bold after webmentions code was added](https://trello.com/c/NmryEL6O/170-oops-about-and-author-pages-are-all-bold-after-webmentions-code-was-added)

File: content/_meta/info/legal.njk [hunk 1]
```diff
@@ -8,7 +8,7 @@ est: |
 
 {{ est | md | safe }}
 
-<p class="h-card">
+<p class="h-card p-org">
```
Comment (jgerigmeyer) [inline content/_meta/info/legal.njk:11]: I'm not sure this is doing anything, since it's already a `p` (and won't be bold)?

**Review by jgerigmeyer — APPROVED** [review]

File: content/_meta/info/legal.njk (hunk 1 above)
Comment (mirisuzanne) [reply content/_meta/info/legal.njk:11]: That's part of the h-card microformat for organizations. I didn't add it for the sake of style, but for the sake of completing the pattern.

File: src/scss/initial/_type.scss [hunk 2]
```diff
@@ -39,9 +39,9 @@ hr {
 
 // H-Cards
 // -------
-/// By default, non-link h-cards are bold.
+/// By default, non-link/non-org inline h-cards are bold.
 /// @group type
-.h-card:not([href]) {
+span:where(.h-card):not(.p-org) {
```
Comment (stacyk) [inline src/scss/initial/_type.scss:44]: loving these `:where`, `:has`, etc type of pseudos.

**Review by stacyk — APPROVED** [review]
Looks great!

**Pull request merged**